source/vocab/display.jsonld

```diff
@@ -231,6 +231,12 @@
           "@type": "fresnel:Lens",
           "classLensDomain": "TableOfContents",
           "showProperties": [ "label" ]
+        },
+        "marc:EnumeratedTerm": {
+          "@id": "EnumeratedTerm-chips",
+          "@type": "fresnel:Lens",
+          "classLensDomain": "EnumeratedTerm",
```

Comment:
classLensDomain here is `marc:EnumeratedTerm`. (The `@id` is local and opaque, and is fine as it is.)